jobs/integr8ly/ocp3/nightly/osd-testing-nightly-trigger.yaml

```diff
@@ -0,0 +1,147 @@
+                                string(name: 'customerAdminPassword', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+                                string(name: 'threeScaleAwsAccessKey', value: "${AWS_ACCESS_KEY_ID}"),
+                                string(name: 'threeScaleAwsSecretKey', value: "${AWS_SECRET_ACCESS_KEY}"),
+                                string(name: 'threeScaleBucketName', value: "${THREESCALE_BUCKET_NAME}"),
+                                string(name: 'threeScaleBucketRegion', value: "${THREESCALE_BUCKET_REGION}"),
+                                string(name: 'backupAwsKey', value: "${AWS_ACCESS_KEY_ID}"),
+                                string(name: 'backupAwsSecret', value: "${AWS_SECRET_ACCESS_KEY}"),
+                                string(name: 'backupsBucketName', value: "${BACKUPS_BUCKET_NAME}"),
+                                string(name: 'clientSecret', value: "${SSO_SECRET}")
+                            ])
+                        }
+                    } // stage
+
+                    stage ('Create test user') {
+                        build(job: 'sso-user-create-tests', propagate: false, parameters: [
+                            string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
+                            string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
+                            string(name: 'CUSTOMER_ADMIN_USERNAME', value: "customer-admin"),
+                            string(name: 'CUSTOMER_ADMIN_PASSWORD', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+                            string(name: 'SSO_URL', value: "${SSO_URL}"),
+                            string(name: 'USERNAME', value: "${USERNAME}"),
+                            string(name: 'NUMBER_OF_USERS', value: "1"),
+                            booleanParam(name: 'CLEAN_RESOURCES', value:  Boolean.valueOf("false")),
+                        ])
+                    } // stage
+
+                /**    stage ('Run tests') {
+                        buildStatus = build(job: 'all-tests-executor', propagate: false, parameters: [
+                            string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
+                            string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
+                            string(name: 'CLUSTER_URL', value: "https://console.${CLUSTER_NAME}.openshift.com"),
+                            string(name: 'EVALS_USERNAME', value: "${USERNAME}01"),
+                            string(name: 'ADMIN_USERNAME', value: "${ADMIN_USERNAME}"),
+                            string(name: 'ADMIN_PASSWORD', value: "${ADMIN_PASSWORD}"),
+                            string(name: 'CUSTOMER_ADMIN_USERNAME', value: "customer-admin"),
+                            string(name: 'CUSTOMER_ADMIN_PASSWORD', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+                            string(name: 'NAMESPACE_PREFIX', value: "openshift-"),
+                            string(name: 'WEBAPP_URL', value: "${WEBAPP_URL}"),
```

Comment:
can webapp url be constructed the same way as cluster url? (e.g. without providing the hardcoded value)?

Reply:
Well yes. We should probably use single style - either hardcode everything in params section or construct it as e.g `CLUSTER_URL`

Reply:
yep +1 for unification. One small advantage of CLUSTER_NAME is that it is much nicer in currentBuild.description than full URL.

Reply:
Not sure if we want to add that currentBuild.description to this nightly trigger. Probably nice to have.